R/store.R

```diff
@@ -64,7 +93,12 @@ file_store <- R6Class(
       dir.create(path_out, showWarnings=FALSE, recursive=TRUE)
       file_copy(file_in, path_out)
     }
-    ))
+  ))
+
+file_info <- function(filename) {
+  info <- file.info(filename, extra_cols = FALSE)
```

Comment:
Does this function support len(filename) != 1? How about renaming to file_mtime_and_size() or similar?